tests/PySys/apt_plugin/apt_sequence_remove_with_version_fail/run.py

```diff
@@ -28,12 +28,12 @@ def setup(self):
 
     def execute(self):
         self.plugin_cmd(
-            "remove", "outp_remove", 5, argument=self.package, version=self.version
+            "remove", "outp_remove", 2, argument=self.package, version=self.version
```

Comment:
Do we know why different code was returned previously or have we captured wrong error code before?

Reply:
Not sure, I checked the PR for this test and even then `exit_code = 5` was mapped to Internal Errors and `exit_code = 2` was mapped to other errors (non-apt_plugin related ones).